contracts/exchange-libs/contracts/src/LibOrder.sol

```diff
@@ -133,6 +133,12 @@ library LibOrder {
         // ));
 
         assembly {
+            // Assert order offset (this is an internal error that should never be triggered)
+            let orderOffset := order
```

Comment:
Why create another variable here? We already reference the offset with just `order` below.

Reply:
Yeah I guess it doesn't really make it more readable. I'll remove.